custom_components/ramses_cc/binary_sensor.py

```diff
@@ -108,20 +108,13 @@ def __init__(
         **kwargs,  # leftover attr_dict values
     ) -> None:
         """Initialize a binary sensor."""
-
-        _LOGGER.info("Found a Binary Sensor for %s: %s", device.id, state_attr)
-
-        super().__init__(
-            broker,
-            device,
-            state_attr,
-            device_class=device_class,
-        )
+        _LOGGER.info("Found %r: %s", device, state_attr)
```

Comment:
I've shortened these logs and standardised on use of RF entity repr as the logger name already indicates type.